### ISSUE TITLE
Disable container move by setting a flag, instead of  using the pmvmAccessCommodity

### ISSUE THREAD

**Issue by songbinliu** [turbonomic/kubeturbo#91]
In current supply chain ( vm -> pod - > container -> app -> vapp), Container entities are bonded to the hosting pod, and should not be moved.  We [implement this constrain by using CmmodityDTO_VMPM_ACCESS](https://github.com/songbinliu/kubeturbo/blob/2aa273514033412f7d1effd2e0a998a63848b6aa/pkg/discovery/dtofactory/container_dto_builder.go#L153).   

TODO: There should be some other way, by setting a flag, to disable the move action for container entities.

**Comment by enlinxu** [comment]
The scope is more than vmpm_access commodity. For daemonSet and static pod, we make the pod not monitored to avoid it to move. We want to address that by using the flag as well.

**Comment by songbinliu** [comment]
For pods created by [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/), should we also disable the move and resize?  Because these kinds of pods are expected to live a short time.  

"Jobs are complementary to Replication Controllers. A Replication Controller manages pods which are not expected to terminate (e.g. web servers), and a Job manages pods that are expected to terminate (e.g. batch jobs)."

**Comment by enlinxu** [comment]
I agree. For pod from Jobs, we can provide value only when we have initial placement support.

**Comment by DongyiYang** [comment]
I assume this flag is not configurable, right? It always disables *move*, *resize* of pods created by Job, DaemonSet. Also, make sure to take a look at Stateful Set, does it make sense to move or resize and whether those actions are disruptive.

**Comment by songbinliu** [comment]
yea, need do more tests about StatefulSet, and have a better understanding of its behaviors.
